apps/passport-client/src/dispatch.ts

```diff
@@ -444,6 +488,14 @@ function userInvalid(update: ZuUpdate) {
   });
 }
 
+function userHasChangedPassword(update: ZuUpdate) {
+  saveAnotherDeviceChangedPassword(true);
+  update({
+    userHasChangedPassword: true,
```

Comment:
This will be set back to false in the `loadInitialState()` function, and the localStorage will be cleared on logout (which is the "reset-passport" redux action)